tests/utils/coordinates.test.js

```diff
@@ -8,7 +8,7 @@ suite('utils.coordinates', function () {
     });
 
     test('rejects invalid coordinate', function () {
-      assert.ok(coordinates.isCoordinate('1 1 2.5 -3'));
+      assert.ok(!coordinates.isCoordinate('1 1 2.5 -3'));
```

Comment:
Yes, this test was incorrect. It was not testing for a rejected bogus coordinate